src/lib/stories.ts

```diff
@@ -414,15 +415,19 @@ const parseDateComparator: (arg: string) => (date: string) => boolean = arg => {
     };
 };
 
-const checkoutStoryBranch = (story: StoryHydrated, prefix: string = '') => {
+const buildStoryBranch = (story: StoryHydrated, prefix: string = '') => {
     prefix = prefix || `${config.mentionName}/ch${story.id}/${story.story_type}-`;
     let slug = story.name
         .toLowerCase()
         .replace(/\s/g, '-')
         .replace(/[^a-z0-9-]/g, '')
         .slice(0, 30)
         .replace(/-$/, '');
-    const branch = `${prefix}${slug}`;
+    return `${prefix}${slug}`;
+}
+
+const checkoutStoryBranch = (story: StoryHydrated, prefix: string = '') => {
+    const branch = buildStoryBranch(story, prefix);
```

Comment:
This just moved the branch building logic into a separate method to allow it to be used for both the formatter and the checkout logic